src/Accounts/Account.jsx

```diff
@@ -0,0 +1,50 @@
+import React, {useState, useEffect} from "react";
```

Comment:
why is this component's folder name is `Accounts` and component file name is `Account`?